README.md

```diff
@@ -11,8 +11,8 @@ backend to run as a system service) as well as eos-kolibri-tools.
 
 Provides dbus and systemd configuration to integrate the Kolibri flatpak with
 Endless OS. It creates a `kolibri` system user, and dbus system service
-configuration for `org.learningequality.Kolibri.Daemon`. This dbus service uses
-the `org.learningequality.Kolibri` flatpak.
+configuration for `org.endlessos.Key.Daemon`. This dbus service uses
+the `org.endlessos.Key` flatpak.
```

Comment:
Can you explain here that this is now controllable through a build-time option, which defaults to `org.endlessos.Key`?

Reply:
Good question!  This part is blocked on [Publish the Endless Key app on Flathub](https://github.com/endlessm/endless-key-flatpak/issues/5), actually.  Maybe, I should split commit ("[build: Set kolibri_flatpak_id's default value as org.endlessos.Key](https://github.com/endlessm/eos-kolibri/pull/21/commits/143bc97c3d7913d788aaa38b7c57f05ab2a0651b)") into another PR.

Reply:
Splitted commit ("[build: Set kolibri_flatpak_id's default value as org.endlessos.Key](https://github.com/endlessm/eos-kolibri/pull/21/commits/143bc97c3d7913d788aaa38b7c57f05ab2a0651b)") into [build: Set kolibri_flatpak_id's default value as org.endlessos.Key](https://github.com/endlessm/eos-kolibri/pull/22)

Reply:
Sorry, I think I wasn't clear here. What I meant was, that the README file should say that this app ID and D-Bus name are configurable at build time, instead of simply changing its value in the README. So in the future, people working on this code base learn that information from the README.

Reply:
Oh!  Got it!  Updated as the new commit.